models/transaction.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * This file contains wrapper functions to interact with the DB.
+ * This will contain the DB schema if we start consuming an ORM for managing the DB operations
+ */
+
+const firestore = require('../utils/firestore')
+const transactionsRef = firestore.collection('transaction')
+
+/**
+ * fetch latest 10 transactions from DB from specific userId provided
+ * @param userId { String }: User Id String to be used to fetch latest transactions
+ * @returns {Promise<{isUserExist: boolean, transactions: []}>}
+ */
```

Comment:
I guess we shouldn't hardcode the value `10`
We could pass query params for the number of transactions required. Default value can be 10 though.